tardis/opacities/opacity_state_numba.py

```diff
@@ -10,75 +9,105 @@
 C_SPEED_OF_LIGHT = const.c.to("cm/s").value
 
 
-opacity_state_spec = [
-    ("electron_density", float64[:]),
-    ("t_electrons", float64[:]),
-    ("line_list_nu", float64[:]),
-    ("tau_sobolev", float64[:, :]),
-    ("transition_probabilities", float64[:, :]),
-    ("line2macro_level_upper", int64[:]),
-    ("macro_block_references", int64[:]),
-    ("transition_type", int64[:]),
-    ("destination_level_id", int64[:]),
-    ("transition_line_id", int64[:]),
-    ("bf_threshold_list_nu", float64[:]),
-    ("p_fb_deactivation", float64[:, :]),
-    ("photo_ion_nu_threshold_mins", float64[:]),
-    ("photo_ion_nu_threshold_maxs", float64[:]),
-    ("photo_ion_block_references", int64[:]),
-    ("chi_bf", float64[:, :]),
-    ("x_sect", float64[:]),
-    ("phot_nus", float64[:]),
-    ("ff_opacity_factor", float64[:]),
-    ("emissivities", float64[:, :]),
-    ("photo_ion_activation_idx", int64[:]),
-    ("k_packet_idx", int64),
-]
+@jitclass
+class OpacityStateNumba:
+    electron_density: nb.float64[:]  # type: ignore[misc]
```

Comment:
Not sure what those comments are for

Reply:
to make sure the lint4er doesn't go nuts